### DISCUSSION TITLE
KNX Wetterstation - Lichemesswert und Windmesswert

### DISCUSSION THREAD

**Discussion by Jeremy2403** [brutella/hkknx-public#276] [Q&A]
Hallo, ich bin Anfänger. Ich habe mit der Testversion alle Geräte ausprobiert und habe nun folgende Frage: Warum kann ich zwar Temperaturwerte meiner KNX Wetterstation auslesen und in HomeKit auch angezeigt bekommen, aber keine Werte für Lich (Helligkeit) und Windstärke?

**Comment by brutella** [comment]
Für die Helligkeit kannst du einen Lichtsensor in hkknx erstellen.

Für die Windstärke gibt es in HomeKit keine offiziell Unterstützung. 🤷‍♂️

**Comment by Jeremy2403** [comment]
Danke für die Antwort. Kann man mit der home+ 6 App die Windstärke darstellen? Gesendet mit der WEB.DE iPhone App

**Reply by brutella** [reply]
Ja, es wäre möglich, würde aber gegen die Philosophie von hkknx gehen, dass alle Funktionen über die Apple Home-App abgerufen werden können.

Mir ist die Limitierungen von HomeKit in Verbindung mit der Apple Home-App bewusst, und arbeite an einer Lösung.
